.github/workflows/ci.yaml

```diff
@@ -3,25 +3,23 @@ name: myCI
 on:
   #workflow_dispatch: #manual start from Github UI or API
   push:
-    branches: master
-  #pull_request:
-    #branches: master
+    branches: [master, dev]
+  pull_request:
+    branches: [master, dev]
```

Comment:
so it is dangerous to run this pipeline. dev/master should be separate pipelines

Reply:
done